framework/dotnet/BenchPress.TestFramework/Program.cs

```diff
@@ -1,2 +1,37 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿using System.Collections.Generic;
+
+namespace BenchPress.TestFramework.Azure
+{
+    public static class ResourceGroup
```

Comment:
Fully static classes and methods is a code smell to me that we could be designing this differently. For example, in this case, why not have a resource group object that contains metadata such as the name and the config object and then methods on that that allow for checking if it exists? Or another implementation if you prefer having statics around is to utilize extension methods

Reply:
we tried a few options in a parking lot and decided to have a signature like this. I'm open to suggestions, though.

Reply:
I think this approach leaks implementation logic....It seems we are optimizing the C# framework to work best for powershell when it should be best in class for C# as well! I was not around for the parking lot discussion, would be great to understand the reasoning here and discuss it further